examples/browser-add/index.js

```diff
@@ -11,24 +11,25 @@ function store () {
     }
 
     res.forEach(function (file) {
-      console.log('successfully stored', file.Hash)
-      display(file.Hash)
+      if (file && file.hash) {
+        console.log('successfully stored', file.hash)
+        display(file.hash)
+      }
     })
   })
 }
 
 function display (hash) {
-  ipfs.cat(hash, function (err, res) {
+  ipfs.cat(hash, {buffer: true}, function (err, res) {
```

Comment:
I still think this `buffer: true` is one of the most misleading features. Can you add a comment that says:
"Buffer the whole response instead of streaming it"